ci/pipeline.yml

```
@@ -83,17 +94,150 @@ jobs:
+    ensure:
+      put: destroy-devstack
+      resource: gcp-devstack
+      params:
+        terraform_source: bosh-openstack-cpi-release/ci/terraform/ci/gcp-devstack
+        action: destroy
+      get_params:
+        action: destroy
+
+- name: run-devstack-tests-ruby-cpi
+  serial: true
+  serial_groups: [ devstack ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)